sysutils/libvirt/Portfile

```diff
@@ -3,17 +3,18 @@
 PortSystem          1.0
 PortGroup           compiler_blacklist_versions 1.0
 PortGroup           legacysupport 1.0
+PortGroup           meson 1.0
 
 # O_CLOEXEC
 legacysupport.newest_darwin_requires_legacy 10
 
 # Remember to update libvirt and py-libvirt at the same time.
```

Comment:
Please update py-libvirt to the same version.

Reply:
Will do, I noticed that comment and made a mental note to update py-libvirt as well but as we can see it did not happen :).